source/blog/2019-07-18-release-notes.md

```diff
@@ -0,0 +1,145 @@
+    underling c.t.finagle.Http.Client. The new HttpClientModuleTrait does not provide any
+    default bindings, so it is up to users to supply them - this allows for custom binding
+    annotations and binding multiple HttpClients, which was not previously possible with
+    HttpClientModule. [fe0c94aa](https://github.com/twitter/finatra/commit/fe0c94aaa616c0f1a63516a3278437ac549850c3)
+
+    To migrate,
+
+    ``` 
+    class MyHttpClientModule extends HttpClientModule {
+      override val dest = "flag!mydest"   
+      override val sslHostname = Some("sslHost")
+    }
+    ```
+
+    becomes
+
+    ```
+    class MyHttpClientModule extends HttpClientModuleTrait {
+      override val dest = "flag!mydest"
+      override val label = "myhttpclient"
+      val sslHostname = "sslHost"  
+      // we only override in this example for TLS configuration with the `sslHostname`
+      override def configureClient(
+        injector: Injector,
+        client: Http.Client
+      ): Http.Client = client.withTls(sslHostname)
+       
+      @Singleton
+      @Provides
+      final def provideHttpClient(
+        injector: Injector,
+        statsReceiver: StatsReceiver,
+        mapper: FinatraObjectMapper
+      ): HttpClient = newHttpClient(injector, statsReceiver, mapper)
+
+      // Note that `provideHttpClient` no longer needs an injected `Service[Request, Response]` so
+      // the following is only needed if you require a `Service[Request, Response]` elsewhere:
+
+      @Singleton
+      @Provides
+      final def provideHttpService(
+        injector: Injector,
+        statsReceiver: StatsReceiver
+      ): Service[Request, Response] = newService(injector, statsReceiver)
+
+    } 
+    ```
+
+### [Scrooge](https://github.com/twitter/scrooge/) ###
+
+-   scrooge-generator: A deprecated \$FinagleClient constructor which does not
+    use RichClientParam has been removed. [28061ba9](https://github.com/twitter/scrooge/commit/28061ba9a6879e80946c1ae0a46b0e255fef2372)
+
+### [Twitter Server](https://github.com/twitter/twitter-server/) ###
+
+#### Changes
+
+-   Remove c.t.server.util.TwitterStats as it is dead code. [55d6d288](https://github.com/twitter/twitter-server/commit/55d6d28862f7f260f3171342ad8ca363553bac40)
+
+### [Util](https://github.com/twitter/util/) ###
+
+#### Breaking API Changes
+
+-   util-core: Removed deprecated c.t.concurrent.Scheduler methods usrTime,
+    cpuTime, and wallTime. These were deprecated in 2015 and have no
+    replacement. [0d77572c](https://github.com/twitter/util/commit/0d77572c76c7c54c0b10a1d25856af16148fe3c4)
+-   util-core: Removed deprecated com.twitter.logging.config classes SyslogFormatterConfig,
+    ThrottledHandlerConfig, SyslogHandlerConfig. These were deprecated in 2012 and have
+    no replacement. Users are encouraged to use 'util-slf4j-api' where possible. [28d9de59](https://github.com/twitter/util/commit/28d9de59eea2adeac82691a57c1954dffb9596fa)
+
```

Comment:
```suggestion

#### Runtime Behavior Changes

-   util-core: Remove experimental toggle `com.twitter.util.BypassScheduler` used
    for speeding up `ConstFuture.map` (`transformTry`). Now, we always run map
    operations immediately instead of via the Scheduler, where they may be queued
    and potentially reordered. [64114324](https://github.com/twitter/util/commit/64114324d8d0d36725cfddac28e99840077f036f)
    
```